CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.14.0] - 2020-04-24
+
```

Comment:
Technically this is missing a Changed header :-p

Reply:
Fixed, as I had to fix merge conflicts here anyway.